package.json

```diff
@@ -47,7 +47,7 @@
     "long": "^5.2.3",
     "protobufjs": "^7.2.4",
```

Comment:
Should this be removed if we're not using protobuffs anymore?

Reply:
Yes, good catch. Removed this and also `ts-proto`.